frb_codegen/src/generator/dart/mod.rs

```diff
@@ -63,7 +63,10 @@ pub fn generate(ir_file: &IrFile, config: &Opts, wasm_funcs: &[IrFuncDisplay]) -
         ..
     } = &spec;
     let needs_freezed = spec.needs_freezed;
-    let common_header = generate_common_header();
+    let mut common_header = generate_common_header();
+    if config.no_use_bridge_in_method {
+        common_header.import += "import 'ffi.io.dart' if (dart.library.html) 'ffi.web.dart';";
+    }
```

Comment:
uber-nit: shall we put this `if` into `generate_common_header`?

Reply:
This is bad because there is no config information.
generate_common_header actually gets an initial value, so there should be nothing wrong with that.
Initial value, and then processing.
If you add config, it doesn't mean and initial value.

But I can change it if you want-_-||

Reply:
Btw, with the suggestion below, maybe just do `let common_header = generate_common_header() + config.extra_headers;`